### PR TITLE
Updated distribution build path in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mikalv**
Updated incorrect path for distribution files in CONTRIBUTING.md.

### CODE REVIEW

**Comment by mikalv** [comment]
Hm, the CLA is signed, did it in front of creating PR.

**Comment by tvernum** [comment]
@elasticmachine OK to test

**Comment by tvernum** [comment]
Team: @elastic/es-core-infra

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ cd elasticsearch/
 ./gradlew assemble
 ```
 
-You will find the newly built packages under: `./distribution/(deb|rpm|tar|zip)/build/distributions/`.
+You will find the newly built packages under: `./distribution/(packages|archives)/(deb|rpm|tar|zip)/build/distributions/`.
```
Comment (jasontedor) [inline CONTRIBUTING.md]: We probably need to be clearer here. Like separate the tar and zip with archives and rpm and deb with packages. As is, it’s ambiguous.

**Review by jasontedor — CHANGES_REQUESTED** [review]
Thanks for the PR, this is in the right direction. I left a suggestion.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mikalv) [reply CONTRIBUTING.md]: Yea, you're right. I'll fix.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -162,7 +162,12 @@ cd elasticsearch/
 ./gradlew assemble
 ```
 
-You will find the newly built packages under: `./distribution/(deb|rpm|tar|zip)/build/distributions/`.
+For package managers like dpkg and rpm you will find the newly built packages under:
+`./distribution/packages/(deb|rpm)/build/distributions/`
+
+Regular compressed archives like tar and zip is found under:
```
Comment (jasontedor) [inline CONTRIBUTING.md]: “is” -> “are”

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -162,7 +162,12 @@ cd elasticsearch/
...
+`./distribution/packages/(tar|zip)/build/distributions/`
```
Comment (jasontedor) [inline CONTRIBUTING.md]: “packages” -> “archives”

**Review by jasontedor — COMMENTED** [review]
Thanks. I left some minor nits. Also, sorry for the duplicate comment. I did not realize I was on the commit versus the pull request.

File: CONTRIBUTING.md (hunk 3 above)
Comment (mikalv) [reply CONTRIBUTING.md]: Sorry about that, both fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mikalv) [reply CONTRIBUTING.md]: Corrected.

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by jasontedor** [comment]
Thanks a lot @mikalv.

**Comment by mikalv** [comment]
No problem, happy to help out on great projects :)

**Pull request merged**